README.md

```diff
@@ -83,14 +83,26 @@ Latest commit deployed to PlayStore: 792 from Nov 6th 11AM
 
 <br>
 
+
 ## How to run the project
+- Make sure you have installed flutter and all the requirements
+	- [Official flutter installation guide](https://docs.flutter.dev/get-started/install)
+- Currently the app uses `3.0.5` flutter version. So open the terminal and change the directory to flutter location and run the following commands
+    - `git checkout 3.0.5` 
```

Comment:
Isn't `flutter downgrade 3.0.5` the suggested way to do this

Reply:
For downgrade command i was getting error, saying
"There is no previously recorded version for channel stable" so i added the generalized solution for it ! Git checkout won't fail in anyone's system!

Reply:
Perhaps but you should mention the "I'm lucky=I started working on flutter more than a year ago" version, aka "flutter downgrade".
And keep your "git checkout" option in case it failed.

Reply:
Will do that :p